bika/lims/content/sample.py

```diff
@@ -760,13 +747,19 @@ def setClientReference(self, value, **kw):
             ar.Schema()['ClientReference'].set(ar, value)
         self.Schema()['ClientReference'].set(self, value)
 
+    def getClientReference(self):
+            return self.Schema().getField('ClientReference').get(self)
```

Comment:
Bad indentation?

Reply:
Anyhow, since there is a `ClientReference` field defined in the Schema already, why the creation of a getter is still required (plone automatically generates the getter for each Schema field at runtime)?